src/renderers/controls/time.control.tsx

```diff
@@ -0,0 +1,58 @@
+import { JSX } from '../JSX';
+import * as _ from 'lodash';
+import { and, formatIs, RankedTester, rankWith, uiTypeIs } from '../../core/testers';
+import { Control, ControlProps, ControlState } from './Control';
+import {
+    formatErrorMessage,
+    mapStateToControlProps,
+    registerStartupRenderer
+} from '../renderer.util';
+import { connect, Event } from '../../common/binding';
+
+/**
+ * Default tester for time controls.
+ * @type {RankedTester}
+ */
+export const timeControlTester: RankedTester = rankWith(2, and(
+    uiTypeIs('Control'),
+    formatIs('time')
+));
+
+export class TimeControl extends Control<ControlProps, ControlState> {
+
+    render() {
+        const { classNames, id, visible, enabled, errors, label, uischema } = this.props;
+
+        const isValid = errors.length === 0;
+        const divClassNames = 'validation' + (isValid ? '' : ' validation_error');
+
+        return (
+            <div className={classNames.wrapper}>
+                <label htmlFor={id} className={classNames.label} data-error={errors}>
```

Comment:
Sorry, I just now saw this. As far as I know the data-error is attribute is not defined for a html label, and is a materialize only attribute,

Reply:
you may be right, but all the other non-material controls have it as well. Shall I remove it from every non-material control, then?

Reply:
no, just from the ones you touch

Reply:
@edgarmueller or should we do a dedicated issue out of this?

Reply:
Yes, I've done so, see [here](https://github.com/eclipsesource/jsonforms/issues/707).

Reply:
@edgarmueller @eneufeld Ok, thanks for #707. So I won't touch it here to avoid conflicts with the solution of #707